### PR TITLE
feat: update new client generation to include rest/rpc documentation …

### PR DESCRIPTION
**Pull request by alicejli**
…links

https://github.com/googleapis/google-cloud-java/issues/9930

### CODE REVIEW

File: generation/new_client/README.md [hunk 1]
```diff
@@ -172,17 +172,23 @@ Example: `https://cloud.google.com/alloydb/docs`
 
 ### REST Docs
 
-The corresponding value in the Cloud Drop page is `documentation_uri`.
+Not every library has these, as they are generated adhoc. Poke around the product docs link to see if there is a valid REST API reference link.
```
Comment (suztomo) [inline generation/new_client/README.md:175]: Would you avoid starting a section with a negative statement? (It sounds like similar to starting with a section with "Note that ...")

File: generation/new_client/README.md [hunk 2]
```diff
@@ -172,17 +172,23 @@ Example: `https://cloud.google.com/alloydb/docs`
...
-Example: `https://cloud.google.com/alloydb/docs`
+Example: `https://cloud.google.com/alloydb/docs/reference/rest`
+
+If they exist, add them as a flag to the python command below like:
+`--rest-docs="https://cloud.google.com/alloydb/docs/reference/rest" \`
 
 ### RPC Docs
 
-The corresponding value in the Cloud Drop page is `documentation_uri`.
+Not every library has these, as they are generated adhoc. Poke around the product docs link to see if there is a valid RPC API reference link.
```
Comment (suztomo) [inline generation/new_client/README.md:185]: Would you avoid starting a section with a negative statement? (It sounds like similar to starting with a section with "Note that ...")

**Review by suztomo — CHANGES_REQUESTED** [review]

File: generation/new_client/README.md [hunk 3]
```diff
@@ -172,17 +172,25 @@ Example: `https://cloud.google.com/alloydb/docs`
 
 ### REST Docs
 
-The corresponding value in the Cloud Drop page is `documentation_uri`.
+This link must be found manually (until b/310975675 is implemented), and may not exist for every library, as they are generated adhoc.
```
Comment (suztomo) [inline generation/new_client/README.md:175]: Avoid starting a paragraph with "Note that..." sentence.

File: generation/new_client/README.md [hunk 4]
```diff
@@ -172,17 +172,25 @@ Example: `https://cloud.google.com/alloydb/docs`
...
-Example: `https://cloud.google.com/alloydb/docs`
+Example: `https://cloud.google.com/alloydb/docs/reference/rest`
+
+If they exist, add them as a flag to the python command below like:
+`--rest-docs="https://cloud.google.com/alloydb/docs/reference/rest" \`
 
 ### RPC Docs
 
-The corresponding value in the Cloud Drop page is `documentation_uri`.
+This link must be found manually (until b/310975675 is implemented), and may not exist for every library, as they are generated adhoc.
```
Comment (suztomo) [inline generation/new_client/README.md:186]: Avoid starting a paragraph with "Note that..." sentence. Start with a sentence of how to get this value.

Comment (alicejli) [reply generation/new_client/README.md]: Updated as there's now an official Cloud Drop field for it

File: generation/new_client/README.md (hunk 3 above)
Comment (alicejli) [reply generation/new_client/README.md]: Updated as there's now an official Cloud Drop field for it

**Review by suztomo — APPROVED** [review]

File: generation/new_client/README.md [hunk 5]
```diff
@@ -172,17 +172,23 @@ Example: `https://cloud.google.com/alloydb/docs`
 
 ### REST Docs
 
-The corresponding value in the Cloud Drop page is `documentation_uri`.
+The corresponding value in the Cloud Drop page is `rest_reference_documentation_uri`.
 The value must starts with "https://".
 
-Example: `https://cloud.google.com/alloydb/docs`
+Example: `https://cloud.google.com/alloydb/docs/reference/rest`
```
Comment (suztomo) [inline generation/new_client/README.md:178]:
```suggestion
Example: `https://cloud.google.com/alloydb/docs/reference/rest` in https://github.com/googleapis/googleapis/blob/master/google/cloud/alloydb/v1/alloydb_v1.yaml
```

But I didn't find the value in the YAML file. Is this a good example?

**Review by suztomo — CHANGES_REQUESTED** [review]
Can we have the example of REST and RPC document with a Cloud Drop data?

**Review by suztomo — APPROVED** [review]

**Pull request merged**